stubs/2.7/collections.pyi

```diff
@@ -1,4 +1,119 @@
+# Stubs for collections
+
+# Based on http://docs.python.org/3.2/library/collections.html
+
+# TODO UserDict
+# TODO UserList
+# TODO UserString
+# TODO more abstract base classes (interfaces in mypy)
+
 # NOTE: These are incomplete!
 
 # namedtuple is special-cased in the type checker; the initializer is ignored.
 namedtuple = object()
+
+from typing import (
+    TypeVar, Iterable, Generic, Iterator, Dict, overload,
+    Mapping, List, Tuple, Callable, Set, Sequence, Sized,
+    Optional, Union
+)
+import typing
+
+_T = TypeVar('_T')
+_KT = TypeVar('_KT')
+_VT = TypeVar('_VT')
+
+
+# namedtuple is special-cased in the type checker; the initializer is ignored.
+namedtuple = object()
```

Comment:
Duplicate definition of `namedtuple`. You should be able to catch errors like these by using something like `mypy --py2 -m collections`.